### DISCUSSION TITLE
SFMTA's GTFS

### DISCUSSION THREAD

**Discussion by acnetj** [transitland/transitland#408] [Q&A] [answered]
SFMTA's GTFS can be found here: http://api.511.org/transit/datafeeds?api_key=XXXXXXX&operator_id=SF

API key can be obtained here: https://511.org/open-data/token

Andy
Transit.wiki

**Comment by drewda** [comment, marked as answer]
Thanks for pointing this out. Transitland currently imports SFMTA data via the SF Bay Area Regional GTFS feed: https://www.transit.land/feeds/f-sf~bay~area~rg That feed is composed of all of the individual feeds served out by api.511.org, so includes the same contents as the URL that you provided.

This does sound confusing to users who just want to use Transitland to find individual agency feeds. We'll update Transitland Atlas to indicate the single-agency 511.org URL, even though Transitland won't actually import it.
